### DISCUSSION TITLE
SOG Praire Fire Compat Missing?

### DISCUSSION THREAD

**Discussion by The-Pink-Panther** [acemod/ACE3#8583] [Q&A] [answered]
With the latest ace update, 3.14.0, both GM and SOG Praire Fire were getting ace compats. These aren't included in ace, when loading up SOG Prairie Fire you get an error message: 
![image](https://user-images.githubusercontent.com/75080046/138495050-4cce1881-4c8f-4f99-9f45-7e54fada130d.png)

Looking at ACE 3's steam workshop you see that the SOG and GM compats are not there:
![image](https://user-images.githubusercontent.com/75080046/138495188-84b4bc96-6cfc-4239-bca3-acbfe2176ab8.png)

Figured with 3.14.0 releasing, so too would the compats.

**Comment by jonpas** [comment, marked as answer]
Go to main ACE item and click the link in the description.

They will be published openly in a bit.

**Reply by BlueSky-fur** [reply]
It's been 4 years🤔 any news on this at all? 

Been a looooooong time since I last played, but I can't find it in the workshop either.

**Reply by Drofseh** [reply]
> It's been 4 years🤔 any news on this at all?
> 
> Been a looooooong time since I last played, but I can't find it in the workshop either.

There are no seperate compats anymore. They are included inside the main Ace mod now.
